### ISSUE TITLE
DisplayItems attribute name is returning None

### ISSUE THREAD

**Issue by guriosam** [Galarzaa90/tibia.py#26]
I was querying the items of a character at the bazaar and the DisplayItem attribute name is returning None

Here is my code snippet:

```
response : TibiaResponse = self.client.loop.run_until_complete(self.client.fetch_auction(character_auction_id, 
        fetch_items=True))
        
details : AuctionDetails = response.data

items : List[DisplayItem] = details.items.entries

print(items)
```

The print of the returned list:

`
[<DisplayItem name=None count=22 item_id=238>, <DisplayItem name=None count=347 item_id=239>, <DisplayItem name=None count=11 item_id=266>]
`

Yesterday everything was working fine, the problem emerged this morning. Maybe the Tibia Bazaar website changed something in their HTML indexing?

**Comment by Galarzaa90** [comment]
Could be, sometimes silent changes are introduced that break functionality.

**Comment by Galarzaa90** [comment]
@guriosam I could not replicate, could you tell me the specific auction id used?

**Comment by guriosam** [comment]
Weird, its happening for all auction ids i am trying here.
But take few examples:

- 329473
- 327345
- 316107
- 331201

Also, I did a little debugging here and the problem is on the function __parse_image_box_ for the DisplayItem class at bazaar,py file.
Specifically on this line: `m = description_regex.search(title_text)`, it is returning None.

**Comment by Galarzaa90** [comment]
I found the issue, they changed the way the tooltips are displayed:

Before: `22x "a mana potion"`
After: `22x mana potion`

So it is not parsing them correctly because the format doesn't match anymore. I will push a fix later.

**Comment by guriosam** [comment]
Awesome, thank you Garlaza!

**Comment by Galarzaa90** [comment]
Should be fixed in v3.7.0
